traffic-control-openshift/README.md

```diff
@@ -1,25 +1,61 @@
 # Traffic Control on OpenShift
 This demo demonstrates how to use traffic control to test applications running in [OpenShift](https://www.openshift.com/). It uses OpenShift version 3 to run the Kubernetes cluster, [tcd](https://github.com/kinvolk/tcd) to do traffic control on the Kubernetes nodes and [Weave Scope](https://github.com/weaveworks/scope) to monitor, visualize and interact with the Kubernetes cluster.
 
-## What do you need to start?
+To find more information about traffic control and this demo check our [blog post](https://kinvolk.io/blog/2016/05/testing-web-services-with-traffic-control-on-kubernetes/).
+
+## Prepare for the demo
+### What do you need to start?
 - Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You will need to reboot the VM after the first `vagrant up` in order to load the right kernel version and to load the modules needed by tcd.
```

Comment:
the sentence "You will need to reboot..." with the explanation could be moved below where you explain how to reboot, since this section is just about listing the repositories that you need.